site/static/docs/5.1/assets/img/boosted-sprite.svg

```diff
@@ -40,5 +40,7 @@
   </symbol>
   <symbol fill="currentColor" viewBox="0 0 30 30" id="document">
     <path fill-rule="evenodd" d="M18.75 2.25v4.5c0 .828.597 1.5 1.333 1.5h4.667l-6-6zm1.5 7.5c-1.657 0-3-1.343-3-3v-4.5h-12v24c0 .828.672 1.5 1.5 1.5h18v-18h-4.5zm-10.5 1.5h10.5c.414 0 .75.336.75.75s-.336.75-.75.75H9.75c-.414 0-.75-.336-.75-.75s.336-.75.75-.75zm0 3h10.5c.414 0 .75.336.75.75s-.336.75-.75.75H9.75c-.414 0-.75-.336-.75-.75s.336-.75.75-.75zm0 3h10.5c.414 0 .75.336.75.75s-.336.75-.75.75H9.75c-.414 0-.75-.336-.75-.75s.336-.75.75-.75zm3 4.5h-3c-.414 0-.75-.336-.75-.75s.336-.75.75-.75h3c.414 0 .75.336.75.75s-.336.75-.75.75z"/>
+  <symbol fill="currentColor" viewBox="0 0 30 30" id="add">
```

Comment:
I think it should be removed because already embedded for the Quantity Selector directly

Reply:
Done